subsys/fs/ext2/ext2_format.c

```diff
@@ -72,24 +67,24 @@ static void set_bitmap_bits(uint8_t *bitmap, uint32_t to_set)
 }
 
 static void default_directory_inode(struct ext2_disk_inode *in, uint32_t nblocks,
-					struct ext2_cfg *cfg)
+				    struct ext2_cfg *cfg)
 {
 	LOG_DBG("Set directory inode: %p", in);
-	in->i_mode       = sys_cpu_to_le16(EXT2_DEF_DIR_MODE);
-	in->i_uid        = 0;
-	in->i_size       = sys_cpu_to_le32(nblocks * cfg->block_size);
-	in->i_atime      = 0;
-	in->i_ctime      = 0;
-	in->i_mtime      = 0;
-	in->i_dtime      = 0;
-	in->i_gid        = 0;
-	in->i_blocks     = sys_cpu_to_le32(nblocks * cfg->block_size / 512);
-	in->i_flags      = 0;
-	in->i_osd1       = 0;
+	in->i_mode = sys_cpu_to_le16(EXT2_DEF_DIR_MODE);
+	in->i_uid = 0;
+	in->i_size = sys_cpu_to_le32(nblocks * cfg->block_size);
+	in->i_atime = 0;
+	in->i_ctime = 0;
+	in->i_mtime = 0;
+	in->i_dtime = 0;
+	in->i_gid = 0;
+	in->i_blocks = sys_cpu_to_le32(nblocks * cfg->block_size / 512);
+	in->i_flags = 0;
+	in->i_osd1 = 0;
 	in->i_generation = 0;
-	in->i_file_acl   = 0;
-	in->i_dir_acl    = 0;
-	in->i_faddr      = 0;
+	in->i_file_acl = 0;
+	in->i_dir_acl = 0;
+	in->i_faddr = 0;
```

Comment:
Looks like you've misconfigured your editor to format all lines in the file, and not only the lines you've changed. 

Please revert all format changes that are not changed